### PR TITLE
Add `gc-` as `git checkout -`

### PR DESCRIPTION
**Pull request by xnuk**

### CODE REVIEW

**Comment by simnalamburt** [comment]
이거 어떻게 쓰는건가여?

**Comment by xnuk** [comment]
`-` 쓰면 이전에 있던 브랜치를 가리켜요

**Comment by xnuk** [comment]
<blockquote><dl>
<dt>&lt;branch&gt;</dt>
<dd>

Branch to checkout; if it refers to a branch (i.e., a name that, when prepended with "`refs/heads/`", is a valid ref), then that branch is checked out. Otherwise, if it refers to a valid commit, your `HEAD` becomes "detached" and you are no longer on any branch (see below for details).

You can use the `@{-N}` syntax to refer to the `N`-th last branch/commit checked out using "`git checkout`" operation. **You may also specify `-` which is synonymous to `@{-1}`.**

As a special case, you may use `A...B` as a shortcut for the merge base of A and B if there is exactly one merge base. You can leave out at most one of A and B, in which case it defaults to `HEAD`.

</dd></dl></blockquote>

**Comment by simnalamburt** [comment]
혹시 `gc-` 는 별로인가염 argument에 `-`가 들어가는 커맨드들이 좀 더 있을거같아성

**Comment by xnuk** [comment]
조아요 오홓홓

File: abbreviations [hunk 1]
```diff
@@ -198,3 +198,4 @@ grhd                 git reset HEAD
 grhhd                git reset --hard HEAD
 grhh1                git reset --hard HEAD~1
 gsta                 git stash
+g-                   git checkout -
```
Comment (simnalamburt) [inline abbreviations]:
```suggestion
gc-                   git checkout -
```

File: abbreviations [hunk 2]
```diff
@@ -198,3 +198,4 @@ grhd                 git reset HEAD
 grhhd                git reset --hard HEAD
 grhh1                git reset --hard HEAD~1
 gsta                 git stash
+gc-                   git checkout -
```
Comment (simnalamburt) [inline abbreviations]:
```suggestion
gc-                  git checkout -
```

**Pull request merged**